src/components/AddRpcButton/hooks/useAddRpcWithTimeout.ts

```diff
@@ -0,0 +1,53 @@
+import { useCallback } from "react";
+import { addRpcUrlAction as sendToAnalytics, AddRpcUrlActionType } from "@src/lib/analytics/events";
+import { ERROR_ADD_MANUALLY_MESSAGE } from "@src/utils/handleRpcError";
+import { useAddRpcEndpoint } from "@src/hooks/useAddRpcEndpoint";
+import { WalletClient } from "viem";
+import { AddToWalletStateValues } from "..";
```

Comment:
Do we use `;` in this project? Or is it just the editor adding it because it like it?

Reply:
I think the lint setup has them. We could remove, i don't like them either